compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -1223,4 +1224,26 @@ object messages {
            |Attempting to define a field in a method signature after a varargs field is an error.
            |""".stripMargin
   }
+
+  case class AmbiguousOverload(tree: tpd.Tree, alts: List[SingleDenotation], pt: Type)(
+    err: typer.ErrorReporting.Errors)(
+    implicit ctx: Context)
+  extends Message(AmbiguousOverloadID) {
+
+    private def all = if (alts.length == 2) "both" else "all"
+
+    override def msg: String =
```

Comment:
These shouldn't be `override`, I'm guessing you looked at the one above and just assumed that we write it like this now. Could you change both? 😄